### ISSUE TITLE
[!! important] Application of WCAG criteria in Checklists for Audio and Video and in WCAG Standard

### ISSUE THREAD

**Issue by mmeller75** [w3c/wai-media-guide#121]
Checklists for Audio and Video include Captions (AA) for Live Audio-only.
The tables in the WCAG Standard section include:
-  A 1.2.3 for Video only
-  AA 1.2.4 for Audio only

Is this correct? Reading the criteria, both [1.2.3](https://www.w3.org/TR/WCAG21/#audio-description-or-media-alternative-prerecorded) and [1.2.4](https://www.w3.org/TR/WCAG21/#captions-live) specifically reference **synchronized media**. Video-only and audio-only would not be synchronized "with another format for presenting information and/or with time-based interactive components", so these two SCs don't seem to apply. [SC 1.2.9](https://www.w3.org/TR/WCAG21/#audio-only-live), which would include captions for live audio-only, is AAA.

Unless this resource is applying them as best practice. If so, can that be made more clear? Including descriptive audio for video-only is a great recommendation, but the way the tables are set up, it is implied that this is required.

**Comment by shawna-slh** [comment]
Thanks for the input @mmeller75 !

I'll ask AG folks to review this again.

Reviewers note:
* [Checklists](https://www.w3.org/WAI/media/av/planning/#checklist) - include both:
   * What is required in WCAG at Level A, AA, and AAA.
   * What is needed to meet user needs, beyond WCAG. (If it doesn’t have any ‘A’, then it is not required in WCAG.)
* [WCAG Standard section](https://www.w3.org/WAI/media/av/planning/#wcag-standard) - should be only what is **required** in WCAG.

**Comment by alastc** [comment]
The [Time based media SC](https://www.w3.org/TR/WCAG21/#time-based-media) for 1.2.2 through to 1.2.7 are specific to 'synchronised media', which is generally synchronised audio & video, although it could be audio + text or video + text (or something else).

Only 1.2.1, 1.2.8 & 1.2.9 apply to video and/or audio only.

So if I'm understanding the aim of the [checklists](https://www.w3.org/WAI/media/av/planning/#checklist) properly:
- Live Audio-only, Captions should not show the (AA).
- Under "Pre-Recorded Video", the "Does the video have visual information" list, those apply when the video has synchronised audio (or synchronised something).

In the pre-recorded table under the "[WCAG Standard](https://www.w3.org/WAI/media/av/planning/#wcag-standard)" heading:
- The  video-only Audio desc cell shouldn't include 1.2.3 & 1.2.5.

Pinging @bruce-usab as someone that knows the details of this better than I do for confirmation...

**Comment by mmeller75** [comment]
@alastc In addition to what you listed above, as I understand it, the Live table under the "WCAG Standard" heading shouldn't list 1.2.4 in the Audio-only Captions cell.

**Comment by bruce-usab** [comment]
Thanks for looping me in.  Is there a GitHub resource I might edit to show what I think the changes should be?

For example, at AAA, my understanding is that 1.2.9 Audio-only (Live) requires live streaming text, and that a static  transcript in that case is _not_ sufficient.

Another AAA example (1.2.8 Media Alternative (Prerecorded)) is that the checklist asserts, that for multimedia (a.k.a, video with speech or other meaningful audio), the “Transcript of audio information [can be] The same text from the captions file, in a different format.”

In my experience, this does not work!  The CC timed text file will make use of the synchronization, and so might not identify that the person talking has changed.  (Live captioning will indicate speaker changes with >> but even good CC might skip that if the speaker change is obvious from the onscreen visual action.)  Also, CC will not repeat text that appears on screen (e.g., name placard of a speaker) and that sort of detail very much needs to in a transcript.

**Comment by shawna-slh** [comment]
> Is there a GitHub resource I might edit to show what I think the changes should be?

Yes. This is the repo. Would be great to get your suggested edits. Thanks much!

Here is the page:
   https://github.com/w3c/wai-media-guide/blob/master/planning.md

You can fork, edit, and submit a pull request.
(You can even see how your changes will look in the rendered page. Once you submit the pull request, several seconds later there will be a preview available -- as [explained here](https://www.w3.org/wiki/WAI/Translation_Instructions#Preview).)

**Comment by bruce-usab** [comment]
Sorry to say I won’t get to this until early next week.

Do you prefer a single set of suggested edits (i.e., one pull request for all my changes) or should there be several pull requests, with one suggested change per pull request?

(I have no idea what pull requests look like on the owner’s side.  Maybe this is a nonsensical question!)

**Comment by shawna-slh** [comment]
It's a good question. In this case, you can put them all in the same pull request -- will be easier to preview.

**Highest priority is to get the standards requirements fixed** in the tables and checklists.

I can take your input on the CC files needing more to be good transcripts and integrate it. So you don't need to write that if you don't want to.

**Comment by bruce-usab** [comment]
I finished editing my fork and submitted the [pull request](https://github.com/w3c/wai-media-guide/pull/122).

Resource implies that captions are provided for audio descriptions.  That does happen sometimes, but the practice is counter-productive.

SC 1.2.5 does not apply to video-only media because it is not synchronized media.

SC 1.2.8 Media Alternative (Prerecorded) does not apply to audio-only media because it is not synchronized media.

Among other edits:

- added “screenplay” wherever “descriptive transcript” appeared
- changed “standards” to “requirements”
- changed “description” to “audio description” most places
- changed inch mark / double prime to curly quotes because neutral quote marks are ugly and amateurish

Edit 10/2:  I am happy to provide line-by-line rational for my edits if that helps.

**Comment by bruce-usab** [comment]
@slhenry what is timeline for getting my edits incorporated (and/or debated)?

**Comment by shawna-slh** [comment]
HI Bruce. Thanks for the ping. I had missed your "I finished editing my fork" message.

I put this on my To Do list with HIGH priority. Possibly can get to it this week. For sure by early next week.

~Shawn

**Comment by shawna-slh** [comment]
Summary of changes:
* WCAG requirements in 4 sections, relating to 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.8
* deleted “Transcript of audio information [can be] The same text from the captions file, in a different format.”.<br>added: “Often you'll need to add visual information to the transcript, such as text that is in the video.”
* deleted “of the description itself” in "Develop **[captions](/media/av/captions/)** for the main video, for the described version, and of the description itself."

Changes are shown in this [pull request](https://github.com/w3c/wai-media-guide/pull/123/files)<br>and listed in the [changelog]( https://deploy-preview-123--wai-media-guide.netlify.com/media/av/changelog/).

<hr>

Other change suggestions:
> added “screenplay” wherever “descriptive transcript” appeared

I’d like to think about that some more, yet haven’t had the bandwidth. I created a [separate issue]( https://github.com/w3c/wai-media-guide/issues/124) for it.

> changed “standards” to “requirements”

“Standards” is used based on input from usability testing with people who are not familiar with WCAG. If you feel strongly about this change, maybe most efficient if we chat briefly?

> changed “description” to “audio description” most places

EOWG spent quite a bit of effort considering this and doing (informal) research on [Terminology - Audio Description / Video Description / Described Video #75](https://github.com/w3c/wai-media-guide/issues/75). I’d prefer not to re-open it.

> changed inch mark / double prime to curly quotes because neutral quote marks are ugly and amateurish

I created a [WAI-site-wide issue](https://github.com/w3c/wai-website/issues/199) for it.

<hr>

I think I caught everything. If I missed something, please open a new issue.

Thanks, all!

**Comment by yatil** [comment]
@bruce-usab 

> changed inch mark / double prime to curly quotes because neutral quote marks are ugly and amateurish

FWIW our Markdown parser already does that conversation, so the output should always have curly quotes (and other typographic symbols) even if the source text has none. (Many people are not used to typing them and it is quite hard to type them in certain operating systems.)
